src/app.jsx

```diff
@@ -108,6 +108,12 @@ class AppActive extends React.Component {
             resourceHasError: {},
             notificationIdCnt: 0,
             path: cockpit.location.path,
+            /* virt-install feature support checks */
+            cloudInitSupported: undefined,
```

Comment:
We could put all virt-install related booleans into a structure, then passing them around is more generic.  Like
```
virtInstallFeatures: {
   available: undefined,
   cloudInitSupport: undefined,
   ...
}
```

Reply:
ah sure - let me repost then.

Reply:
Created the struct when passing to the child component. For having the struct like this already in the state I feel it makes the code uglier, as each of these properties are updates independently.

Reply:
Actually, it is possible to run `virt-install --install=? --cloud-init=? --unattended=?`, but that makes the parsing more difficult, I guess.

Reply:
This last command won't work because it early fails if ones of these is missing.
That means that I need to update the state like this for each of the calls:
 this.setState({ virtInstallFeatures: { ...this.state.virtInstallFeatures, downloadOSSupported: true } }), ..

This seems actually uglier than the initial state - don't you think?

Reply:
Yes, let's not bother with this. :-)